torchtext/datasets/squad1.py

```diff
@@ -30,5 +30,5 @@
 @_wrap_split_argument(('train', 'dev'))
 def SQuAD1(root, split):
     extracted_files = download_from_url(URL[split], root=root, hash_value=MD5[split], hash_type='md5')
```

Comment:
Is it file or files?

Reply:
sorry about this naming issue, updated it already. It is just a single file downloaded. Note that download_from_url returns path to single downloaded file.